components/Layout/Header/Header.tsx

```diff
@@ -71,17 +30,24 @@ const HeaderComponent = ({
             </a>
           </div>
 
-          <nav className="lbh-header__links" aria-label="Navigation menu">
-            {navLinks && (
-              <>
-                {navLinks.map(({ name, path }) => (
-                  <Link href={path} key={path}>
-                    <a className="govuk-header__link">{name}</a>
-                  </Link>
-                ))}
-              </>
-            )}
-          </nav>
+          {user && (
+            <nav className="lbh-header__links" aria-label="Navigation menu">
+              <Link href="/search">
+                <a className="govuk-header__link">Search</a>
+              </Link>
+              <Link href="/">
+                <a className="govuk-header__link">My work space</a>
+              </Link>
+              {canUserManageWorkers(user) && (
+                <Link href="/workers">
+                  <a className="govuk-header__link">Manage workers</a>
+                </Link>
+              )}
```

Comment:
When I hard code the URL as an adult user, I can get to the manage worker's page. 
Would we need to have some redirect for this use case too?

Reply:
@Renny4Real yeah good question – at the moment none of our routes are protected from people hard coding the URL. I think we _should_ change that, but I think that's beyond the scope of this PR